### ISSUE TITLE
Select and loop

### ISSUE THREAD

**Issue by giankam** [nulloy/nulloy#75]
It would be great to select just a part of a song and loop it.

**Comment by MegaLoler** [comment]
Yeah, me too, I use Audacity for listening to snippets of audio at a time, but I would much prefer to just have a normal player that can do that instead of relying on a full editing program.
